docs/azure_jumpstart_arcbox/ITPro/_index.md

```diff
@@ -446,6 +446,50 @@ After deployment is complete, its time to start exploring ArcBox. Most interacti
 
   ![Screenshot showing ArcBox Client VM with Hyper-V](./hypervterminal.png)
 
+- Alternately, you can use Azure CLI to [connect to Azure Arc-enabled servers using SSH](https://learn.microsoft.com/azure/azure-arc/servers/ssh-arc-overview?tabs=azure-cli).
+
+  - Windows (PowerShell)
+
+    ```powershell
+    az login
+
+    $rgName = "arcbox-rg"
+    $serverName = "ArcBox-Win2K22"
+    $localUser = "arcdemo"
+
+    az ssh arc --resource-group $rgName --name $serverName --local-user $localUser
+    ```
+
+  - Linux (shell)
+
+    ```shell
+    az login
+
+    export rgName="arcbox-rg"
+    export serverName="ArcBox-Win2K22"
+    export localUser="arcdemo"
+
+    az ssh arc --resource-group $rgName --name $serverName --local-user $localUser
+    ```
```

Comment:
```suggestion
- Alternately, you can use Azure CLI to connect to one of the Azure Arc-enabled servers, Hyper-V Ubuntu virtual machines [using SSH](https://learn.microsoft.com/azure/azure-arc/servers/ssh-arc-overview?tabs=azure-cli). Open a PowerShell session and use the below commands.

  ```powershell
  az login -u $env:SPN_CLIENT_ID -p $env:SPN_CLIENT_SECRET -t $env:SPN_TENANT_ID

  $serverName = "ArcBox-Ubuntu-01"
  $localUser = "arcdemo"

  az ssh arc --resource-group $Env:resourceGroup --name $serverName --local-user $localUser
  ```
```

Reply:
a couple more things I caught:

- The first command is exclusive to connecting to the Ubuntu machines, so it needs to be clear.
- I removed the shell code block since it's irrelevant. The user will be connecting from the client machine via PowerShell unless they will install WSL, which is out of scope of our automation.
- The name in the _$serverName_ env var needs to be _ArcBox-Ubuntu-01_ since again, the user does not connect here to the Windows VM.
- For the _az login_ command, I suggest using the SPN-related env vars we inject into the Client VM as part of ArcBox automation.
- We already have _$Env:resourceGroup_ env var injected to the Client VM as part of ArcBox automation, so no need for the user to create a new var.
- Upon initial command run, I have been asked to install the _az ssh_ extension. To avoid this, this CLI extension needs to be installed as part of the overall automation in the _ArcServersLogonScript_ PoSH script.
![image](https://user-images.githubusercontent.com/46730876/213960463-a1b9c16b-84d5-49ad-bcb8-656b8d447054.png)

Reply:
Valid points, we should test inside the ArcBox VM - although I have been testing from my local machine in order to try connecting to the VMs from the outside.

I tested logging in using the service principal now. The environment variables is correct, but it fails due to lack of _--service-principal_. I have added that parameter in a new commit and verified it works.

> Upon initial command run, I have been asked to install the az ssh extension. To avoid this, this CLI extension needs to be installed as part of the overall automation in the ArcServersLogonScript PoSH script.
Yes, we should avoid this prompt. Another option to consider is the dynamic installation of extensions as described in the [documentation](https://learn.microsoft.com/en-us/cli/azure/azure-cli-extensions-overview#how-to-install-extensions):

> When you run an extension command that is not installed, the Azure CLI can recognize the command you run, and automatically install the extension for you 

This might be beneficial in case there are other scenarios in the future which requires extensions. Or do we want to explicitly install the extensions we know we will need?